homeassistant/components/amcrest/camera.py

```diff
@@ -388,6 +406,21 @@ def _enable_audio(self, enable):
         else:
             self._audio_enabled = enable
             self.schedule_update_ha_state()
+        if self._control_light:
+            self._enable_light(self._audio_enabled or self.is_streaming)
+
+    def _enable_light(self, enable):
+        """Enable or disable indicator light."""
+        from amcrest import AmcrestError
```

Comment:
You can now import things on top, out of functions

Reply:
Cool! I rebased, moved the imports, tested and force pushed.

Reply:
@pvizeli is this ok now and ready for a merge, or does it still require more review? Thanks!